start.bat

```diff
@@ -106,10 +106,15 @@ for /f "delims=" %%i in ('dir *runtime-decompiler-*.jar /B /S') do set JRD=%%i
 
 popd
 
+rem Maps an available drive letter if script is located on a network drive, does pretty much nothing otherwise - done to counteract CMD not supporting UNC paths
+pushd %PORTABLE_JRD_HOME%
+
 rem Create environment variable pointing to script's location
-set "PROPERTY_LOCATION=-Djrd.location=%PORTABLE_JRD_HOME%"
 set "PROPERTY_PURPOSE=-Djrd.purpose=%PURPOSE%"
+set "PROPERTY_LOCATION=-Djrd.location=%CD%"
```

Comment:
afaik this can not work. CD is working dir, not JRD's dir

Reply:
pushd %PORTABLE_JRD_HOME% makes it work

Reply:
> Sorry, I case to disagre with htmlzie of the license. Please ping petra and radek, what they thought
> -- Mgr. Jiri Vanek judovana@email.cz
> ---------- Původní e-mail ---------- Od: Prokop Tunel <notifications@github.com> Komu: pmikova/java-runtime-decompiler <java-runtime-decompiler@noreply. github.com> Datum: 12. 8. 2019 9:58:51 Předmět: Re: [pmikova/java-runtime-decompiler] Overall fixes (#53) " @AurumTheEnd commented on this pull request. ---------------------------------------------------------------------------- In LICENSE ([#53 (comment)](https://github.com/pmikova/java-runtime-decompiler/pull/53#discussion_r312810225)) :
> @@ -1,53 +1,62 @@
> - GNU GENERAL PUBLIC LICENSE - Version 3, 29 June 2007 - - Copyright (C) 2007 Free Software Foundation, Inc. <http://fsf.org/> +<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"> The window looked bad, asymmetric and the text didn't resize along with it. — You are receiving this because you commented. Reply to this email directly, view it on GitHub (#53?email_source=notifications&email_token=AAWFCS3QFKCEBH5ZLUZ5RRLQEEJ3LA5CNFSM4IH5TAU2YY3PNVWWK3TUL52HS4DFWFIHK3DMKJSXC5LFON2FEZLWNFSXPKTDN5WW2ZLOORPWSZGOCBHAKDA#discussion_r312810225) , or mute the thread (https://github.com/notifications/unsubscribe-auth/AAWFCSZ577J2W72D4U2RNXDQEEJ3LANCNFSM4IH5TAUQ) . "

Have you seen how much better it looks? This is meant to be a cosmetic change.